server/raft.go

```diff
@@ -418,12 +420,17 @@ func (s *Server) startRaftNode(cfg *RaftConfig) (RaftNode, error) {
 		for index := state.FirstSeq; index <= state.LastSeq; index++ {
 			ae, err := n.loadEntry(index)
 			if err != nil {
-				n.warn("Could not load %d from WAL [%+v] with error: %v", index, state, err)
-				continue
+				n.warn("Could not load %d from WAL [%+v]: %v", index, state, err)
```

Comment:
These are pretty scary warnings to users, can we add some indication about what they can do about them? Or mention if the server will recover on its own? 

Seeing just these in the server log and no indication of what action to take is not a good experience.  For myself even, I have no idea what to do about these log lines

Reply:
OK, but like this log says "ask Derek" :) if that's the intended outcome, LGTM :)